chain/block.go

```diff
@@ -609,7 +609,12 @@ func (b *StatelessBlock) innerVerify(ctx context.Context, vctx VerifyContext) er
 	heightKeyStr := string(heightKey)
 	timestampKeyStr := string(timestampKey)
 	feeKeyStr := string(feeKey)
-	tsv := ts.NewView(set.Of(heightKeyStr, timestampKeyStr, feeKeyStr), map[string][]byte{
+
+	heightKeyKey := state.NewKey(heightKeyStr, state.Read, state.Write)
+	timestampKeyKey := state.NewKey(timestampKeyStr, state.Read, state.Write)
+	feeKeyKey := state.NewKey(feeKeyStr, state.Read, state.Write)
```

Comment:
The naming for these variables is weird and it's only referenced in the construction of `NewView` so i would just inline them all like 

```
tsv := ts.NewView(
    set.Of(
	state.NewKey(heightKeyStr, state.Read, state.Write),
	state.NewKey(timestampKeyStr, state.Read, state.Write),
	state.NewKey(feeKeyStr, state.Read, state.Write),
    )
...
)
```